xword_dl/downloader/derstandarddownloader.py

```diff
@@ -0,0 +1,87 @@
+import datetime
+import json
+import urllib
```

Comment:
Needs an `import re` between these two lines. Looks like that got clipped during review, sorry!